internal/core/api.rs

```diff
@@ -101,6 +101,21 @@ impl From<WindowRc> for Window {
     }
 }
 
+#[derive(Copy, Clone, Debug, PartialEq, strum::EnumString, strum::Display)]
```

Comment:
```suggestion
#[derive(Copy, Clone, Debug, PartialEq)]
```

Since this is not used by slint, we don't need the conversion to and from string